### DISCUSSION TITLE
I can't change the font (from Segoe UI to Segoe UI Bold)

### DISCUSSION THREAD

**Discussion by victorhugo1733** [zBritva/charticulator-visual-community#30] [General]
How to change the font (from Segoe UI to Segoe UI Bold)?
It lets me select the font but the change is not made

**Comment by brunoluan** [comment]
Same here.... this is a life savior visual, but this is kind of annoying

**Comment by zBritva** [comment]
@victorhugo1733  @brunoluan Thanks for reporting. There is general bug with font selector. Will include the fix to next version

**Reply by victorhugo1733** [reply]
Thanks to you

**Reply by brunoluan** [reply]
Thank you so much for such a great job!!!

**Comment by samnseir** [comment]
Segoe UI Bold works well in Editor visual, however in View version it seems to fallback to default (Times New Roman?).

**Comment by zBritva** [comment]
@samnseir it's so strange, because fix is not reached to end users, even Editor version. [There is current state of the visuals](https://ilfat-galiev.im/docs/charticulator/changelog)
